src/main/app/src/Components/Landscape/Search/Search.tsx

```diff
@@ -54,56 +64,122 @@ const Search: React.FC<PropsInterface> = ({ setSidebarContent, ...props }) => {
   const [results, setResults] = useState<IItem[]>([]);
   const [facets, setFacets] = useState<IFacet[]>([]);
   const [searchTerm, setSearchTerm] = useState('');
-  const [hasChange, setHasChange] = useState(false);
+  const [searchSupport, setSearchSupport] = useState<any>(null);
+  const [render, setRender] = useState<boolean>(false);
   const classes = useStyles();
   const componentClasses = componentStyles();
   const searchInput = React.useRef<HTMLDivElement>(null);
 
-  const search = useCallback((searchTerm: string, identifier: string) => {
+  /**
+   * Search on search term change, set results.
+   */
+  useEffect(() => {
     if (searchTerm.length < 2) return;
-    setHasChange(false);
+
     get(
       '/api/landscape/' +
         identifier +
         '/search/' +
         encodeURIComponent(searchTerm)
           .replace(/[!'()]/g, escape)
           .replace(/\*/g, '%2A')
-    ).then((result) => {
-      setResults(result);
-    }).catch(reason => {
-      console.warn(reason);
-    });
-  }, []);
+    )
+      .then((result) => {
+        setResults(result);
+      })
+      .catch((reason) => {
+        console.warn(reason);
+      });
+  }, [searchTerm, identifier]);
+
+  /**
+   * Initial loading of facets
+   */
+  useEffect(() => {
+    const facetsHtml = facets.map((facet: IFacet) => (
+      <Box key={facet.dim}>
+        <Typography variant={'h6'}>{facet.dim}</Typography>
+        {facet.labelValues.map((lv) => (
+          <Chip
+            onClick={(e) => {
+              let current = searchInput.current;
+              if (!current) return;
+              if (searchTerm.indexOf(facet.dim + ':' + lv.label) === -1) {
+                setSearchTerm(searchTerm + ' ' + facet.dim + ':' + lv.label);
+                setRender(true);
+              }
+              current.focus();
+            }}
+            variant={'outlined'}
+            size={'small'}
+            key={facet.dim + '' + lv.label}
+            label={lv.label}
+            avatar={<Avatar>{lv.value}</Avatar>}
+          />
+        ))}
+      </Box>
+    ));
 
+    const help = (
+      <div>
+        <h3>Search</h3>
+        Start typing to match items having name parts or description starting with the given term.
+        Find everything starting with 'foo':
+        <br />
+        <pre>foo</pre>
+        <br />
+        Use whitespaces as AND condition (default). Search in name and description for both words:
```

Comment:
What would be the description? 

I tried to select several Wordpress items by typing `wor c` in http://localhost:3000/#/landscape/nivio:example and I was expecting to select wordpress-web and wordpress-redis and wordpress-mariadb, but it only selects wordpress-redis. They belong to the groups"content" and "common".
`wor co` does not find anything, for example. So the first word is the name of the item and the second is?

Reply:
Good point. The "multi field" query regards identifier, name and description. If you want to search in groups, you have to enter "group:co*". In addition, you query translates to "wor* AND c* in the three fields.

That reminds me to add facets for item.type and item.group.

Reply:
I've added #498 as follow-up

Reply:
+ #500 as follow-up